src/amo/components/HomeCarousel/index.js

```diff
@@ -0,0 +1,84 @@
+/* @flow */
+import React from 'react';
+import { compose } from 'redux';
+
+import {
+  ADDON_TYPE_EXTENSION,
+  ADDON_TYPE_THEME,
+  SEARCH_SORT_TOP_RATED,
+} from 'core/constants';
+import translate from 'core/i18n/translate';
+import { visibleAddonType } from 'core/utils';
+import Carousel from 'ui/components/Carousel';
+import CarouselSection from 'ui/components/CarouselSection';
+
+import './styles.scss';
+
+
+type PropTypes = {|
+  i18n: Object,
+|};
+
+export class HomeCarouselBase extends React.Component {
+  props: PropTypes;
+
+  render() {
+    const { i18n } = this.props;
+
+    return (
+      <div className="HomeCarousel">
+        <Carousel
+          sections={[
+            (
+              <CarouselSection
+                key="featured-extensions"
```

Comment:
mmh why is there a `key` here? and why is it the same `key` for all entries?

Reply:
key is required by react for things in arrays that get rendererd in a loop.

Reply:
But this `key` prop should only be added into the loop no?

Reply:
eslint yelled at me if I didn't do it here. 🤷‍♀️

Reply:
oh ok then